src/patternfly/components/InputGroup/input-group-icon.hbs

```diff
@@ -0,0 +1,10 @@
+<div class="pf-c-input-group__icon{{#if input-group-icon--modifier}} {{input-group-icon--modifier}}{{/if}}" 
```

Comment:
do we need this? Why wouldn't you just put an icon in `pf-c-input-group__text`?

Reply:
True its not styled at all, I was thinking in case there is any future styling but we should just add the class when it comes to it

Reply:
need to remove this hbs partial